www/js/lib/uiUtil.js

```diff
@@ -32,6 +32,35 @@ if (webpMachine) {
 }
 
 define(rqDef, function(settingsStore) {
+
+    /**
+     * Displays a Bootstrap alert or confirm dialog box depending on the options provided
+     * @param {*} node 
+     * @param {*} nodeAttribute 
+     * @param {*} content 
+     * @param {*} mimeType 
+     * @param {*} callback 
```

Comment:
You need to add the Types inside the curly brackets and the variable name after. For example:
```
@param {String} label The modal's label or title (if it is a title?)
@param {String} message The message to display to users
```
You only have three variables in the function, so you only need three params.

You also need a return statement:

`@returns {Promise<Boolean>} A Promise for a Boolean that indicates.....`

Reply:
Yes, that is the auto-generated template actually. Left that blank deliberately as mentioned in the Flags too in this PR. Was waiting for approval on the function implementation.

Reply:
Ah, Ok! Sorry, I missed that in the flags.